tensorflow/python/framework/registry.py

```diff
@@ -67,6 +67,13 @@ def register(self, candidate, name=None):
     stack = traceback.extract_stack()
     self._registry[name] = {_TYPE_TAG: candidate, _LOCATION_TAG: stack[2]}
 
+  def list(self):
+      """List registered items.
```

Comment:
hmm... the indentation seems to be 4 spaces here instead of 2. If you fix that, this change should be good to go!